2024-04-03 13:11:51,020 INFO    StreamThr :24598 [internal.py:wandb_internal():87] W&B internal server running at pid: 24598, started at: 2024-04-03 13:11:51.020287
2024-04-03 13:11:51,024 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status
2024-04-03 13:11:51,027 INFO    WriterThread:24598 [datastore.py:open_for_write():85] open: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/run-v3fcfo34.wandb
2024-04-03 13:11:51,028 DEBUG   SenderThread:24598 [sender.py:send():334] send: header
2024-04-03 13:11:51,061 DEBUG   SenderThread:24598 [sender.py:send():334] send: run
2024-04-03 13:11:51,331 INFO    SenderThread:24598 [dir_watcher.py:__init__():216] watching files in: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files
2024-04-03 13:11:51,331 INFO    SenderThread:24598 [sender.py:_start_run_threads():1065] run started: v3fcfo34 with start time 1712171511.019081
2024-04-03 13:11:51,331 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: summary_record
2024-04-03 13:11:51,332 INFO    SenderThread:24598 [sender.py:_save_file():1319] saving file wandb-summary.json with policy end
2024-04-03 13:11:51,334 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: check_version
2024-04-03 13:11:51,335 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: check_version
2024-04-03 13:11:51,399 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: run_start
2024-04-03 13:11:51,496 DEBUG   HandlerThread:24598 [system_info.py:__init__():31] System info init
2024-04-03 13:11:51,496 DEBUG   HandlerThread:24598 [system_info.py:__init__():46] System info init done
2024-04-03 13:11:51,496 INFO    HandlerThread:24598 [system_monitor.py:start():151] Starting system monitor
2024-04-03 13:11:51,496 INFO    SystemMonitor:24598 [system_monitor.py:_start():116] Starting system asset monitoring threads
2024-04-03 13:11:51,497 INFO    HandlerThread:24598 [system_monitor.py:probe():172] Collecting system info
2024-04-03 13:11:51,497 INFO    SystemMonitor:24598 [interfaces.py:start():168] Started cpu
2024-04-03 13:11:51,498 INFO    SystemMonitor:24598 [interfaces.py:start():168] Started disk
2024-04-03 13:11:51,498 INFO    SystemMonitor:24598 [interfaces.py:start():168] Started gpu
2024-04-03 13:11:51,500 INFO    SystemMonitor:24598 [interfaces.py:start():168] Started memory
2024-04-03 13:11:51,501 INFO    SystemMonitor:24598 [interfaces.py:start():168] Started network
2024-04-03 13:11:51,506 DEBUG   HandlerThread:24598 [system_info.py:probe():195] Probing system
2024-04-03 13:11:51,530 DEBUG   HandlerThread:24598 [system_info.py:_probe_git():180] Probing git
2024-04-03 13:11:51,540 DEBUG   HandlerThread:24598 [system_info.py:_probe_git():188] Probing git done
2024-04-03 13:11:51,541 DEBUG   HandlerThread:24598 [system_info.py:probe():241] Probing system done
2024-04-03 13:11:51,541 DEBUG   HandlerThread:24598 [system_monitor.py:probe():181] {'os': 'Linux-5.15.146.1-microsoft-standard-WSL2-x86_64-with-glibc2.35', 'python': '3.9.15', 'heartbeatAt': '2024-04-03T19:11:51.506875', 'startedAt': '2024-04-03T19:11:51.010410', 'docker': None, 'cuda': None, 'args': None, 'state': 'running', 'program': 'src/eda/eda.ipynb', 'root': '/home/kayla/Project-Build-an-ML-Pipeline-Starter', 'git': {'remote': 'https://github.com/kaylacarriger/Project-Build-an-ML-Pipeline-Starter.git', 'commit': '077621b0ed54d88e8c2f5ef3dea7b725c4ea7c98'}, 'email': None, 'host': 'Ardat-Yakshi', 'username': 'kayla', 'executable': '/home/kayla/miniconda3/envs/mlops/bin/python', 'cpu_count': 6, 'cpu_count_logical': 12, 'cpu_freq': {'current': 3293.724999999999, 'min': 0.0, 'max': 0.0}, 'cpu_freq_per_core': [{'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}, {'current': 3293.725, 'min': 0.0, 'max': 0.0}], 'disk': {'total': 1006.853931427002, 'used': 12.122268676757812}, 'gpu': 'NVIDIA GeForce RTX 3050 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3050 Laptop GPU', 'memory_total': 4294967296}], 'memory': {'total': 13.501205444335938}}
2024-04-03 13:11:51,541 INFO    HandlerThread:24598 [system_monitor.py:probe():182] Finished collecting system info
2024-04-03 13:11:51,541 INFO    HandlerThread:24598 [system_monitor.py:probe():185] Publishing system info
2024-04-03 13:11:51,541 DEBUG   HandlerThread:24598 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2024-04-03 13:11:51,542 DEBUG   HandlerThread:24598 [system_info.py:_save_pip():67] Saving pip packages done
2024-04-03 13:11:51,542 DEBUG   HandlerThread:24598 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2024-04-03 13:11:52,333 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_created():275] file/dir created: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/requirements.txt
2024-04-03 13:11:52,334 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_created():275] file/dir created: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/wandb-summary.json
2024-04-03 13:11:52,334 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_created():275] file/dir created: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/conda-environment.yaml
2024-04-03 13:11:53,792 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:11:53,793 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:11:54,334 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/conda-environment.yaml
2024-04-03 13:11:54,426 DEBUG   HandlerThread:24598 [system_info.py:_save_conda():86] Saving conda packages done
2024-04-03 13:11:54,622 DEBUG   HandlerThread:24598 [system_info.py:_save_code():89] Saving code
2024-04-03 13:11:54,622 WARNING HandlerThread:24598 [system_info.py:_save_code():91] unable to save code -- program entry not found
2024-04-03 13:11:54,623 DEBUG   HandlerThread:24598 [system_info.py:_save_patches():127] Saving git patches
2024-04-03 13:11:54,796 DEBUG   HandlerThread:24598 [system_info.py:_save_patches():169] Saving git patches done
2024-04-03 13:11:54,797 INFO    HandlerThread:24598 [system_monitor.py:probe():187] Finished publishing system info
2024-04-03 13:11:54,800 DEBUG   SenderThread:24598 [sender.py:send():334] send: files
2024-04-03 13:11:54,800 INFO    SenderThread:24598 [sender.py:_save_file():1319] saving file wandb-metadata.json with policy now
2024-04-03 13:11:54,801 INFO    SenderThread:24598 [sender.py:_save_file():1319] saving file diff.patch with policy now
2024-04-03 13:11:54,827 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: stop_status
2024-04-03 13:11:54,827 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: stop_status
2024-04-03 13:11:54,956 DEBUG   SenderThread:24598 [sender.py:send():334] send: telemetry
2024-04-03 13:11:55,135 DEBUG   SenderThread:24598 [sender.py:send():334] send: use_artifact
2024-04-03 13:11:55,318 INFO    Thread-17 :24598 [upload_job.py:push():143] Uploaded file /tmp/tmpd2nxd9x6wandb/4ytvnapx-diff.patch
2024-04-03 13:11:55,319 INFO    Thread-16 :24598 [upload_job.py:push():143] Uploaded file /tmp/tmpd2nxd9x6wandb/gdj6jgan-wandb-metadata.json
2024-04-03 13:11:55,335 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_created():275] file/dir created: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/wandb-metadata.json
2024-04-03 13:11:55,335 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_created():275] file/dir created: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/diff.patch
2024-04-03 13:11:55,793 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:11:55,794 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:11:56,270 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:11:56,270 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:11:56,335 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_created():275] file/dir created: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:11:56,781 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:11:58,271 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:11:58,272 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:11:58,337 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:11:58,752 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:11:58,752 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:00,753 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:00,754 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:01,227 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:01,227 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:01,783 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:03,229 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:03,229 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:03,715 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:03,716 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:05,717 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:06,085 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:06,557 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:06,557 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:06,785 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:08,558 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:08,558 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:09,030 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:09,036 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:09,828 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:12:11,037 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:11,619 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:11,787 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:12,090 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:12,090 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:14,091 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:14,091 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:14,565 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:14,566 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:14,830 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:12:16,567 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:16,567 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:16,791 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:17,119 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:17,120 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:19,121 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:19,122 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:19,596 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:19,597 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:19,833 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:12:21,598 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:21,598 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:21,798 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:22,128 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:22,128 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:22,348 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/config.yaml
2024-04-03 13:12:24,129 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:24,130 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:24,600 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:24,600 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:24,835 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:12:26,351 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:12:26,602 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:26,602 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:27,073 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:27,074 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:27,662 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:28,351 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:12:29,084 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:29,084 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:29,554 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:29,555 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:29,836 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:12:31,556 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:32,361 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:32,665 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:32,834 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:32,834 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:34,835 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:34,836 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:34,838 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:12:35,309 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:35,310 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:37,310 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:37,311 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:37,668 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:37,791 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:37,792 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:39,793 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:39,793 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:39,839 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:12:40,266 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:40,267 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:42,268 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:42,268 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:42,669 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:42,751 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:44,840 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:12:44,955 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:44,955 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:45,425 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:45,425 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:47,426 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:47,427 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:47,670 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:47,907 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:47,908 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:49,841 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:12:49,909 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:49,910 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:50,381 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:50,381 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:51,502 DEBUG   SystemMonitor:24598 [system_monitor.py:_start():130] Starting system metrics aggregation loop
2024-04-03 13:12:51,503 DEBUG   SenderThread:24598 [sender.py:send():334] send: stats
2024-04-03 13:12:52,382 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:52,383 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:52,858 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:52,858 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:53,505 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:54,842 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:12:54,859 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:54,859 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:55,330 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:55,330 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:57,332 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:57,332 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:57,803 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:57,804 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:12:58,506 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:12:59,805 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:59,806 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:12:59,843 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:00,280 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:00,281 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:02,282 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:02,283 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:02,763 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:02,763 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:03,507 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:04,764 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:04,765 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:04,844 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:05,247 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:05,248 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:07,252 ERROR   gpu       :24598 [interfaces.py:aggregate():146] Failed to serialize metric: division by zero
2024-04-03 13:13:07,254 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:07,254 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:07,723 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:07,724 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:08,508 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:09,725 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:09,725 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:09,846 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:10,198 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:11,314 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:13,315 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:13,316 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:13,509 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:13,793 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:13,794 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:14,847 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:15,795 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:16,381 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:16,849 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:16,849 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:18,510 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:18,850 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:18,850 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:19,323 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:19,324 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:19,848 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:21,324 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:21,325 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:21,504 DEBUG   SenderThread:24598 [sender.py:send():334] send: stats
2024-04-03 13:13:22,170 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:22,171 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:23,699 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:24,172 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:24,172 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:24,648 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:24,648 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:24,850 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:26,649 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:26,650 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:27,117 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:27,117 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:28,700 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:29,118 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:29,119 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:29,591 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:29,592 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:29,851 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:31,592 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:31,593 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:32,070 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:32,070 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:33,702 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:34,801 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:34,801 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:34,855 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:35,272 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:35,273 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:37,274 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:37,275 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:37,812 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:37,813 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:38,703 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:39,814 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:39,814 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:39,857 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:40,288 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:40,288 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:42,289 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:42,290 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:43,424 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:43,424 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:43,705 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:44,859 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:45,425 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:45,426 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:45,906 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:45,906 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:47,911 ERROR   gpu       :24598 [interfaces.py:aggregate():146] Failed to serialize metric: division by zero
2024-04-03 13:13:47,911 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:47,911 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:48,377 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:48,378 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:48,707 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:49,861 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:50,379 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:50,862 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:51,336 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:51,337 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:51,504 DEBUG   SenderThread:24598 [sender.py:send():334] send: stats
2024-04-03 13:13:53,338 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:53,339 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:53,819 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:53,820 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:54,506 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:54,864 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:13:55,821 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:56,157 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:56,628 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:56,628 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:58,629 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:58,630 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:13:59,097 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:59,098 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:13:59,508 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:13:59,866 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:14:01,099 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:01,100 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:02,548 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:02,549 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:04,509 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:14:04,550 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:04,550 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:04,868 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:14:05,025 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:05,025 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:07,026 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:07,027 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:07,496 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:07,497 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:09,498 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:09,498 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:09,524 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:14:09,870 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:14:09,974 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:09,974 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:10,401 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:14:11,975 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:11,976 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:12,402 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:14:12,456 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:12,457 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:14,403 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:14:14,458 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:14,525 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:14:15,288 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:14:15,290 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:15,763 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:15,763 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:16,404 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:14:17,765 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:17,765 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:18,237 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:18,238 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:18,405 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:14:19,692 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:14:19,875 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: keepalive
2024-04-03 13:14:20,239 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:20,240 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: Unknown Error
2024-04-03 13:14:20,712 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:21,425 DEBUG   SenderThread:24598 [sender.py:send():334] send: telemetry
2024-04-03 13:14:21,477 DEBUG   SenderThread:24598 [sender.py:send():334] send: config
2024-04-03 13:14:21,481 DEBUG   SenderThread:24598 [sender.py:send():334] send: exit
2024-04-03 13:14:21,481 INFO    SenderThread:24598 [sender.py:send_exit():557] handling exit code: 0
2024-04-03 13:14:21,481 INFO    SenderThread:24598 [sender.py:send_exit():559] handling runtime: 150
2024-04-03 13:14:21,484 INFO    SenderThread:24598 [sender.py:_save_file():1319] saving file wandb-summary.json with policy end
2024-04-03 13:14:21,484 INFO    SenderThread:24598 [sender.py:send_exit():565] send defer
2024-04-03 13:14:21,484 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:21,485 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 0
2024-04-03 13:14:21,485 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:21,485 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 0
2024-04-03 13:14:21,485 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 1
2024-04-03 13:14:21,486 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:21,486 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 1
2024-04-03 13:14:21,486 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:21,486 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 1
2024-04-03 13:14:21,486 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 2
2024-04-03 13:14:21,486 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:21,487 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 2
2024-04-03 13:14:21,487 INFO    HandlerThread:24598 [system_monitor.py:finish():161] Stopping system monitor
2024-04-03 13:14:21,488 DEBUG   SystemMonitor:24598 [system_monitor.py:_start():137] Finished system metrics aggregation loop
2024-04-03 13:14:21,488 DEBUG   SystemMonitor:24598 [system_monitor.py:_start():141] Publishing last batch of metrics
2024-04-03 13:14:21,489 INFO    HandlerThread:24598 [interfaces.py:finish():175] Joined cpu
2024-04-03 13:14:21,489 INFO    HandlerThread:24598 [interfaces.py:finish():175] Joined disk
2024-04-03 13:14:21,885 ERROR   gpu       :24598 [interfaces.py:monitor():129] Failed to sample metric: No data
2024-04-03 13:14:21,888 ERROR   gpu       :24598 [interfaces.py:aggregate():146] Failed to serialize metric: division by zero
2024-04-03 13:14:21,888 INFO    HandlerThread:24598 [interfaces.py:finish():175] Joined gpu
2024-04-03 13:14:21,888 INFO    HandlerThread:24598 [interfaces.py:finish():175] Joined memory
2024-04-03 13:14:21,889 INFO    HandlerThread:24598 [interfaces.py:finish():175] Joined network
2024-04-03 13:14:21,889 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:21,889 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 2
2024-04-03 13:14:21,889 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 3
2024-04-03 13:14:21,889 DEBUG   SenderThread:24598 [sender.py:send():334] send: stats
2024-04-03 13:14:21,890 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:21,890 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 3
2024-04-03 13:14:21,890 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:21,890 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 3
2024-04-03 13:14:21,890 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 4
2024-04-03 13:14:21,891 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:21,891 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 4
2024-04-03 13:14:21,891 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:21,891 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 4
2024-04-03 13:14:21,891 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 5
2024-04-03 13:14:21,891 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:21,891 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 5
2024-04-03 13:14:21,892 DEBUG   SenderThread:24598 [sender.py:send():334] send: summary
2024-04-03 13:14:21,892 INFO    SenderThread:24598 [sender.py:_save_file():1319] saving file wandb-summary.json with policy end
2024-04-03 13:14:21,892 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:21,892 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 5
2024-04-03 13:14:21,892 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 6
2024-04-03 13:14:21,893 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:21,893 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 6
2024-04-03 13:14:21,893 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:21,893 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 6
2024-04-03 13:14:21,897 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: status_report
2024-04-03 13:14:22,062 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 7
2024-04-03 13:14:22,062 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:22,062 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 7
2024-04-03 13:14:22,062 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:22,062 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 7
2024-04-03 13:14:22,408 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/config.yaml
2024-04-03 13:14:22,408 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:14:22,408 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/wandb-summary.json
2024-04-03 13:14:22,408 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_created():275] file/dir created: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/code/_session_history.ipynb
2024-04-03 13:14:22,408 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_created():275] file/dir created: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/code
2024-04-03 13:14:22,483 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: poll_exit
2024-04-03 13:14:24,047 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 8
2024-04-03 13:14:24,047 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: poll_exit
2024-04-03 13:14:24,047 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:24,047 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 8
2024-04-03 13:14:24,048 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:24,048 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 8
2024-04-03 13:14:24,049 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 9
2024-04-03 13:14:24,049 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:24,049 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 9
2024-04-03 13:14:24,049 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:24,049 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 9
2024-04-03 13:14:24,049 INFO    SenderThread:24598 [dir_watcher.py:finish():362] shutting down directory watcher
2024-04-03 13:14:24,409 INFO    Thread-13 :24598 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:14:24,409 INFO    SenderThread:24598 [dir_watcher.py:finish():392] scan: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files
2024-04-03 13:14:24,409 INFO    SenderThread:24598 [dir_watcher.py:finish():406] scan save: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/wandb-summary.json wandb-summary.json
2024-04-03 13:14:24,410 INFO    SenderThread:24598 [dir_watcher.py:finish():406] scan save: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/requirements.txt requirements.txt
2024-04-03 13:14:24,410 INFO    SenderThread:24598 [dir_watcher.py:finish():406] scan save: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/diff.patch diff.patch
2024-04-03 13:14:24,410 INFO    SenderThread:24598 [dir_watcher.py:finish():406] scan save: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/config.yaml config.yaml
2024-04-03 13:14:24,419 INFO    SenderThread:24598 [dir_watcher.py:finish():406] scan save: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/conda-environment.yaml conda-environment.yaml
2024-04-03 13:14:24,420 INFO    SenderThread:24598 [dir_watcher.py:finish():406] scan save: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/wandb-metadata.json wandb-metadata.json
2024-04-03 13:14:24,424 INFO    SenderThread:24598 [dir_watcher.py:finish():406] scan save: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log output.log
2024-04-03 13:14:24,429 INFO    SenderThread:24598 [dir_watcher.py:finish():406] scan save: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/code/_session_history.ipynb code/_session_history.ipynb
2024-04-03 13:14:24,433 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 10
2024-04-03 13:14:24,433 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:24,433 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 10
2024-04-03 13:14:24,434 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:24,435 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 10
2024-04-03 13:14:24,435 INFO    SenderThread:24598 [file_pusher.py:finish():160] shutting down file pusher
2024-04-03 13:14:24,926 INFO    Thread-19 :24598 [upload_job.py:push():143] Uploaded file /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/requirements.txt
2024-04-03 13:14:24,928 INFO    Thread-21 :24598 [upload_job.py:push():143] Uploaded file /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/conda-environment.yaml
2024-04-03 13:14:24,930 INFO    Thread-20 :24598 [upload_job.py:push():143] Uploaded file /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/config.yaml
2024-04-03 13:14:24,931 INFO    Thread-23 :24598 [upload_job.py:push():143] Uploaded file /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/code/_session_history.ipynb
2024-04-03 13:14:24,949 INFO    Thread-18 :24598 [upload_job.py:push():143] Uploaded file /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/wandb-summary.json
2024-04-03 13:14:24,972 INFO    Thread-22 :24598 [upload_job.py:push():143] Uploaded file /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/files/output.log
2024-04-03 13:14:25,173 INFO    Thread-12 :24598 [sender.py:transition_state():585] send defer: 11
2024-04-03 13:14:25,173 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:25,174 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 11
2024-04-03 13:14:25,174 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:25,174 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 11
2024-04-03 13:14:25,174 INFO    SenderThread:24598 [file_pusher.py:join():165] waiting for file pusher
2024-04-03 13:14:25,174 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 12
2024-04-03 13:14:25,175 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:25,175 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 12
2024-04-03 13:14:25,175 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:25,175 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 12
2024-04-03 13:14:25,404 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 13
2024-04-03 13:14:25,404 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:25,404 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 13
2024-04-03 13:14:25,404 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:25,405 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 13
2024-04-03 13:14:25,405 INFO    SenderThread:24598 [sender.py:transition_state():585] send defer: 14
2024-04-03 13:14:25,405 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: defer
2024-04-03 13:14:25,405 INFO    HandlerThread:24598 [handler.py:handle_request_defer():170] handle defer: 14
2024-04-03 13:14:25,405 DEBUG   SenderThread:24598 [sender.py:send():334] send: final
2024-04-03 13:14:25,406 DEBUG   SenderThread:24598 [sender.py:send():334] send: footer
2024-04-03 13:14:25,406 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: defer
2024-04-03 13:14:25,406 INFO    SenderThread:24598 [sender.py:send_request_defer():581] handle sender defer: 14
2024-04-03 13:14:25,407 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: poll_exit
2024-04-03 13:14:25,407 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: server_info
2024-04-03 13:14:25,407 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: get_summary
2024-04-03 13:14:25,408 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: poll_exit
2024-04-03 13:14:25,408 DEBUG   SenderThread:24598 [sender.py:send_request():361] send_request: server_info
2024-04-03 13:14:25,410 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: sampled_history
2024-04-03 13:14:25,566 DEBUG   HandlerThread:24598 [handler.py:handle_request():144] handle_request: shutdown
2024-04-03 13:14:25,566 INFO    HandlerThread:24598 [handler.py:finish():841] shutting down handler
2024-04-03 13:14:26,408 INFO    WriterThread:24598 [datastore.py:close():299] close: /home/kayla/Project-Build-an-ML-Pipeline-Starter/src/eda/wandb/run-20240403_131151-v3fcfo34/run-v3fcfo34.wandb
2024-04-03 13:14:26,566 INFO    SenderThread:24598 [sender.py:finish():1491] shutting down sender
2024-04-03 13:14:26,566 INFO    SenderThread:24598 [file_pusher.py:finish():160] shutting down file pusher
2024-04-03 13:14:26,614 INFO    SenderThread:24598 [file_pusher.py:join():165] waiting for file pusher
